qtopconf.yaml

```diff
@@ -12,41 +9,80 @@ user_colour_mappings:  # order should be from more generic-->more specific
 # - '\w+00\d': Blue
  - '\w*cms\w*': Red_L
  - '\w*atl\w+': Cyan_L
-wn_labels_max_len: 15
-alt_label_highlight_colours: [White, Blue_L]
+
 remapping:
- - "([A-Za-z]+)([1-9]|[1-9][0-9]|[0-9][0-9][0-9])$": |
-     lambda m: str(m.group(1)+str(int(m.group(2))+250))
+# - "([A-Za-z]+)([1-9]|[1-9][0-9]|[0-9][0-9][0-9])$": |
+#     lambda m: m.group(1)+str(int(m.group(2))+250)
      # e.g. wn100 --> wn350, wn101 --> wn351 and so on
  - 'torvalds(\d+)' : 'tor\1'
  - 'gridlab(\d+)' : 'glab\1'
  - 'gridmon(\d*)' : 'mon\1'
  - '-' : ''  # removes dashes for shortening (can also replace them with something else, e.g. _
 
-# customised end labels
-end_labels:
- '1': ['={__WNID__}']
- '2': ['={_Worker_}', '={__Node__}']
- '3': ['={_Worker_}', '={__Node__}', '={___ID___}']
- '4': ['={________}', '={_Worker_}', '={__Node__}', '={___ID___}']
- '5': ['={__FOR___}', '={__SALE__}', '={_Worker_}', '={__Node__}', '={___ID___}']
- '6': ['={_SPACE__}', '={__FOR___}', '={__SALE__}', '={_Worker_}', '={__Node__}', '={___ID___}']
- '7': ['={__PLACE_}', '={__YOUR__}', '={_ADVERT_}', '={__HERE__}', '={_Worker_}', '={__Node__}', '={___ID___}']
+
+# what parts of qtop to display
+user_display_parts:
+ - job_accounting_summary
+ - wn_occupancy
+ - user_accounts_pool_mappings
+
+#wn_occupancy:
+# - wn id lines
+# - node state
+# - temperature
+# - core user map
+
+wn_occupancy:
+ - wn id lines:
+     max_len: 15
+     # Masking/clipping functionality. How high should the # earliest node number be (e.g. 50), to make the previous WNs vanish
+     min_masking_threshold: 9
+     # Change this to an int if you want a custom cut matrix length
+     user_cut_matrix_width: 0
+     # character used for vertical separator, for the -o switch
+     separator: '|'
+     alt_label_highlight_colours: [White, Blue_L]
+     # customised end labels
+     end_labels:
+       '1': ['={__WNID__}']
+       '2': ['={_Worker_}', '={__Node__}']
+       '3': ['={_Worker_}', '={__Node__}', '={___ID___}']
+       '4': ['={________}', '={_Worker_}', '={__Node__}', '={___ID___}']
+       '5': ['={__FOR___}', '={__SALE__}', '={_Worker_}', '={__Node__}', '={___ID___}']
+       '6': ['={_SPACE__}', '={__FOR___}', '={__SALE__}', '={_Worker_}', '={__Node__}', '={___ID___}']
+       '7': ['={__PLACE_}', '={__YOUR__}', '={_ADVERT_}', '={__HERE__}', '={_Worker_}', '={__Node__}', '={___ID___}']
+ - temperature:
```

Comment:
eithet temperature does not belong to wn_occupancy or, one of the labels is misleading (likely the first one)